src/update-template.js

```diff
@@ -22,6 +22,7 @@ const { TEMPLATE_STATUS_APPROVED } = require('../src/registry');
         const packagePath = myArgs[0];
         const gitHubUrl = myArgs[1];
         const npmUrl = 'https://www.npmjs.com/package/' + myArgs[2];
+        const status = myArgs[3]
```

Comment:
@AmyJZhao , I don't see where you call `src/update-template.js` with a status specified, the only place is https://github.com/adobe/aio-template-submission/pull/273/files#diff-d7a677c9ed6b55c54b4c54d6de22d4c5832d355a6c9a22ffd98de08932aa8074R82 , but you don't set a status there at all 🤔

Reply:
I call `src/update-template.js` with a status specified in this PR #267, so that I can reuse it in the approve and decline workflows. 
https://github.com/adobe/aio-template-submission/pull/267/files#diff-8b48637a2e3fc85ecf622753a2827ad3ff454c69406cd8c49163b8c2acc7383cR91